extensions/wikia/Hydralytics/specials/SpecialAnalytics.php

```diff
@@ -182,8 +182,8 @@ private function analyticsPage() {
 					foreach ($topPages['pageviews'] as $uri => $views) {
 						$newUri = $this->normalizeUri($uri);
 
-						// remove "/wiki/" and underscores from page names
-						$title = str_replace('/wiki/', '', $newUri);
+						// remove language url part and "/wiki/" and underscores from page names
+						$title = preg_replace('|^(/\w+)?/wiki/|', '', $newUri);
```

Comment:
`$wgArticlePath` global variable can be used instead - all links are local to the wiki where we render this special page..

https://github.com/Wikia/app/blob/dev/extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php#L711